### ISSUE TITLE
Adblock message constantly appearing even when adblock is off

### ISSUE THREAD

**Issue by imrolii** [photopea/photopea#3549]
# TLDR
Photopea constantly spamming an adblock prompt even when adblock is disabled.

# Issue Status
![GitHub issue/pull request detail](https://img.shields.io/github/issues/detail/state/photopea/photopea/3549)

(Badge may be slightly delayed)

# The issue in detail

So I was using Photopea for a school project, inside of Brave. 

If you didn't know, Brave has a built-in adblocker.

Up until now, I haven't gotten such message, but now, I get the same message constantly when trying to work. Screenshot:

![Pop-up which displays](https://user-images.githubusercontent.com/79724190/125494365-ef080aa8-79ef-4607-9b5a-9d723e896d7d.png)

So the first thing I did was disable the adblocker (known as Shields within the browser). However, upon doing this, I still got the exact same message.

![Shields off](https://user-images.githubusercontent.com/79724190/125494624-d2cd3eea-8cd7-43d0-8f43-9000fcb7e512.png)

So I had a look on the internet to find out how to whitelist a site in Brave, and found [this](https://github.com/brave/brave-browser/issues/5290#issuecomment-572755768) issue on the Brave repository.

According to the comment, you need to add this to the filters list in brave://adblock, so I did just that. 
> Brave currently does allow for whitelisting via the brave://adblock/ page using adblock syntax:
`@@||ads.example.com/notbanner^$~script`
or entire sites:
`@@||example.com^$document`
if the OP needs to whitelist logmein they should be able to do so (with all subdomains) by adding:
`@@||logmein.com^`

After doing this and refreshing the page, I still get the same result. 

Any way I can fix this?

# What I've done
- [x] Disable Adblock
- [x] Add an exception in adblock settings
- [x] Report an issue via the 'Report a broken site' function in Brave.

# Brave Version Info
![Brave Version](https://user-images.githubusercontent.com/79724190/125495712-0592bfc9-305f-47bd-ba81-12d7f4ad1804.png)

Any help is greatly appreciated!

**Comment by photopea** [comment]
Hi, do you see ads in Photopea? I guess not, which means you are blocking them. We do not provide any service to people, who refuse to pay or to watch ads.

From your description, I think it is clear, that it is a bug in your browser, not in Photopea. Why don't you report it ot the creators of your browser?

I wrote an article about it: https://www.photopea.com/tuts/ads-in-photopea-do-not-work-what-to-do/

**Comment by imrolii** [comment]
Hi there, thanks for the fast response.

I'll read that article now.

Also about blocking ads and no support: I don't *want* to block the ads on photopea, but the browser isn't co-operating at the moment.

I will submit the issue to my browser's github issues page.

**Comment by imrolii** [comment]
Just submitted an issue to my browser's GitHub Repository.

[Here's the link.](https://github.com/brave/brave-browser/issues/16928)

Thank you for your time!

*I'll update this comment with what happens next*

**Comment by photopea** [comment]
I think someone reported it here: https://github.com/brave/brave-browser/issues/16806
